### ISSUE TITLE
Broken @MergeParams() on version 4.13.5

### ISSUE THREAD

**Issue by milewski** [Romakita/ts-express-decorators#331]
The following: 

```ts
@Controller('/user/:id')
@MergeParams() // <-- this is the problem
export class UserController {}
```
Thows this error on version 4.13.5 and above: 
```bash
Class constructor AuthController cannot be invoked without 'new' // observe that the classname is not even related...


TypeError: Class constructor AuthController cannot be invoked without 'new'
at HandlerBuilder.<anonymous> (/project/node_modules/@tsed/src/common/mvc/class/HandlerBuilder.ts:122:51)
at Generator.next (<anonymous>)
at /project/node_modules/tslib/tslib.js:107:75
at new Promise (<anonymous>)
at Object.__awaiter (/project/node_modules/tslib/tslib.js:103:16)
at HandlerBuilder.invoke (/project/node_modules/@tsed/common/lib/mvc/class/HandlerBuilder.js:95:24)
at /project/node_modules/@tsed/src/common/mvc/class/HandlerBuilder.ts:56:22
at Layer.handle [as handle_request] (/project/node_modules/express/lib/router/layer.js:95:5)
at next (/project/node_modules/express/lib/router/route.js:137:13)
at /project/node_modules/@tsed/src/common/mvc/class/HandlerBuilder.ts:179:20
at AuthMiddleware.use (/project/source/middlewares/AuthMiddleware.ts:34:9)
at <anonymous>
at process._tickCallback (internal/process/next_tick.js:182:7)

```

and it works fine on every version below `4.13.5` tested with `4.13.(2,3,4)`

**Comment by Romakita** [comment]
Thanks @milewski for your report ;).
Fix will be released soon.

**Comment by Romakita** [comment]
Fixed 4.13.7
